src/components/index.tests.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import { Faq } from './index'
+
+const demoQuestions = [
+  {
+    summary: 'First',
+    content: 'uno',
+  },
+  {
+    summary: 'Second',
+    content: 'dos',
+  },
+]
+
+// Localized test
+class MyTest extends React.Component<any, any> {
+  render() {
+    return <Faq questions={demoQuestions} />
```

Comment:
And I suppose later we'll just wrap in a fragment like:

```tsx
render() {
  return (
    <>
       <TestFoo />
       <TestBar />
       { /* etc etc */ }
    </>
  )
}

Reply:
Update: I DID test it out also from the monorepo quotequiz (manually). See notes on  corresponding [PR](https://github.com/getethos/ethos/pull/1863)